### PR TITLE
Add WiFi, BT, & GPS for milletwifi

### PR DESCRIPTION
**Pull request by Susurrus**
The patch for the wifi is obviously bad, but not certain how to approach it. I need it for wifi scanning to work at all (driver seems to crash otherwise). IIs this something I need to carry locally or is it fine for now to keep here in near-mainline?

### CODE REVIEW

File: arch/arm/boot/dts/qcom/qcom-apq8026-samsung-milletwifi.dts [hunk 1]
```diff
@@ -310,6 +310,35 @@
...
 
+&modem {
+	mx-supply = <&pm8226_l3>;
+	pll-supply = <&pm8226_l8>;
+
+	status = "okay";
+};
+
+&pronto {
+	status = "okay";
```
Comment (z3ntu) [inline arch/arm/boot/dts/qcom/qcom-apq8026-samsung-milletwifi.dts]: status as last property, so below pinctrl-*

File: arch/arm/boot/dts/qcom/qcom-apq8026-samsung-milletwifi.dts [hunk 2]
```diff
@@ -310,6 +310,35 @@
...
+
+	vddmx-supply = <&pm8226_l3>;
+	vddpx-supply = <&pm8226_l6>;
+
+	pinctrl-names = "default";
+	pinctrl-0 = <&wcnss_pin_a>;
```
Comment (z3ntu) [inline arch/arm/boot/dts/qcom/qcom-apq8026-samsung-milletwifi.dts:324]: pinctrl-0, then pinctrl-names so swap the lines

**Comment by Susurrus** [comment]
Whoops, sorry, should've caught those!

**Comment by Susurrus** [comment]
Anything else needed to merge this?

**Comment by z3ntu** [comment]
Don't think so, it's just busy weeks on my side. :)

File: arch/arm/boot/dts/qcom/qcom-apq8026-samsung-milletwifi.dts [hunk 3]
```diff
@@ -557,6 +586,13 @@
 		drive-strength = <10>;
 		bias-pull-up;
 	};
+
+	wcnss_pin_a: wcnss-pin-active-state {
+		pins =  "gpio40", "gpio41", "gpio42", "gpio43", "gpio44";
```
Comment (z3ntu) [inline arch/arm/boot/dts/qcom/qcom-apq8026-samsung-milletwifi.dts]: fixing double space after `=`

**Comment by z3ntu** [comment]
* Fixed double space after `=` in pinctrl node
* Re-ordered the commits to first have the wcn36xx driver hack, then enable wifi on the device so it's more logical
* Fixed wcn3600a -> wcn3660a in the wcn36xx commit message
* Fixed double `is` in "Enable modem" commit message

**Review by z3ntu — APPROVED** [review]
Thanks!
And sorry for the delay in the review :)

**Pull request merged**